src/antsibull_changelog/cli.py

```diff
@@ -610,6 +617,12 @@ def command_release(args: Any) -> int:
         paths, config, collection_details, args
     )
 
+    if args.update_galaxy_file:
+        if not paths.galaxy_path:
+            LOGGER.error("Cannot find galaxy.yml file in path.")
```

Comment:
I think we should stop execution if this happens. We cannot do what the user asked us to do, so we should stop.
```suggestion
            LOGGER.error("Cannot find galaxy.yml file in path.")
            return 5
```